### PR TITLE
Routing api v2 info

### PR DESCRIPTION
**Pull request by atulkc**
This PR is to add routing api endpoint URL to /v2/info. We have created PR on cf-release (https://github.com/cloudfoundry/cf-release/pull/784) to take care of bosh spec file changes. Changes in that PR are necessary for this one. So please consider these two PRs together.

Regards,
@atulkc and @fordaz

### CODE REVIEW

**Comment by fordaz** [comment]
Any updates on this PR?

**Comment by dieucao** [comment]
It's near the top of our backlog.  We should get to it soon.

File: bosh-templates/cloud_controller_api.yml.erb [hunk 1]
```diff
@@ -187,6 +187,12 @@ uaa:
 hm9000:
   url: <%= p("hm9000.url", "https://hm9000.#{domain}") %>
 
+<% if p("router.enable_routing_api") %>
+routing_api:
+  url: <%= "https://api.#{domain}/routing" %>
```
Comment (Amit-PivotalLabs) [inline bosh-templates/cloud_controller_api.yml.erb]: How is this supposed to work?  Does this imply the routing-api must be colocated with all the CCs?  Won't nginx send all requests to the CC process, how will the routing-api actually get any requests with this URL?

**Comment by atulkc** [comment]
@dieucao thanks!

**Comment by atulkc** [comment]
Closing this PR and submitting a new one.

**Pull request closed without merging**